src/contract.ts

```diff
@@ -32,6 +36,8 @@ export class AbstractContract {
   public static abi: ABIEntity[];
   public static asm: string;
   public static abiCoder: ABICoder;
+  public static debugAsm?: DebugModeAsmWord[];
```

Comment:
Rename debugAsm -> opcodes. There is no debug here. And it's an array.

Let's also change the original DebugModeAsmWord -> OpCode

Reply:
fixed